### PR TITLE
Update to Polkadot-v1.1.0

### LINKED ISSUE #71: Upgrade to Polkadot SDK v1.1.0
EPT is currently up-to-date with the Polkadot v1.0.0 release. The upcoming v1.1.0 release is the first release based on the new Polkadot SDK GitHub mono-repository. It also brings some useful additional functionality to the Substrate FRAME library, notably the SafeMode and TxPause pallets (which we want to integrate in the EPT template).

### PR DESCRIPTION
**Pull request by weezy20**
Update to `release-polkadot-v1.1.0`
Close #71

### CODE REVIEW

File: node/src/service.rs
Comment (stiiifff) [inline node/src/service.rs:1]: @weezy20 Can you double-check if the node-related changes are in-line with the [similar upgrade on Trappist](https://github.com/paritytech/trappist/pull/295/files#diff-594440db8adfc5d1b37711309312bfa06b49e488309186ad062a896778cf9fbf) ?

Comment (weezy20) [reply node/src/service.rs:1]: @stiiifff Will do!

Comment (weezy20) [reply node/src/service.rs:1]: @stiiifff Just finished giving it a second look. LGTM.

**Comment by weezy20** [comment]
@stiiifff A new configuration type on `cumulus_pallet_parachain_system` has appeared under the `parameterized-consensus-hook` feature. 
This is what a sample configuration looks like : 
```rs
type ConsensusHook = cumulus_pallet_aura_ext::FixedVelocityConsensusHook<
		Runtime,
		RELAY_CHAIN_SLOT_DURATION_MILLIS,
		BLOCK_PROCESSING_VELOCITY,
		UNINCLUDED_SEGMENT_CAPACITY,
	>;
```
This appears to be related with async-backing. Curious if we should start including them in the template.

**Review by stiiifff — APPROVED** [review]

**Pull request merged**